### PR TITLE
Parking Folder Specs modifications for the normalized examples

### PR DESCRIPTION
**Pull request by Joudsi**
corrected the files in the Parking folder in the DataModel repo. 

@jmcanterafonseca

### CODE REVIEW

**Comment by Joudsi** [comment]
@jmcanterafonseca 
could you please let me know if this is fine (format wise) ? is it okay now ?!
thanks

File: specs/Parking/ParkingAccess/doc/spec.md [hunk 1]
```diff
@@ -117,20 +121,24 @@ API implementation, you need to use the `keyValues` mode (`options=keyValues`).
 }
 ```
 
-## Examples of use 2 (?options=keyValues simplified representation for data consumers)
+## ### key-value pairs Example
```
Comment (jmcanterafonseca) [inline specs/Parking/ParkingAccess/doc/spec.md]: only ### is needed

File: specs/Parking/ParkingSpot/doc/spec.md [hunk 2]
```diff
@@ -12,109 +12,122 @@ _one_ group.
...
+  - Mandatory. Not nullable (if `location` is not defined).
+
+- `status` : Status of the parking spot from the point of view of occupancy.
+  - Attribute type: [Text](https://schema.org/Text)
+  - Allowed Values: one Of
+    (`occupied`, `free`, `closed`, `unknown`)
+  - Metadata:
+    - `timestamp` : Timestamp which reflects the date when the attribute value was obtained.
+    - Type: [DateTime](https://schema.org/DateTime)
+  - Optional
```
Comment (jmcanterafonseca) [inline specs/Parking/ParkingSpot/doc/spec.md]: wrong nesting level

File: specs/Parking/ParkingSpot/doc/spec.md [hunk 3]
```diff
@@ -12,109 +12,122 @@ _one_ group.
...
+  - `TimeInstant` :
     [Timestamp](https://github.com/telefonicaid/iotagent-node-lib#TimeInstant)
     saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
     to keep backwards compatibility with current FIWARE reference
-    implementations. + Type: [DateTime]((https://schema.org/DateTime). here can
+    implementations.
+  - Type: [DateTime]((https://schema.org/DateTime). here can
```
Comment (jmcanterafonseca) [inline specs/Parking/ParkingSpot/doc/spec.md]: check nesting level

File: specs/Parking/ParkingSpot/doc/spec.md [hunk 4]
```diff
@@ -12,109 +12,122 @@ _one_ group.
...
+  - Attribute type: [Text](https://schema.org/Text)
+  - Allowed values:
         -   `onstreet` : The parking spot belongs to an onstreet parking site.
         -   `offstreet` : The parking spot belongs to an onstreet parking site.
         -   Other values as per application needs
-    -   Mandatory
+  - Mandatory
 
--   `TimeInstant` :
+- `TimeInstant` :
```
Comment (jmcanterafonseca) [inline specs/Parking/ParkingSpot/doc/spec.md]: in general, please check nesting level of metadata

**Comment by Joudsi** [comment]
is it okay now ? 
@jmcanterafonseca

**Comment by jmcanterafonseca** [comment]
LGTM, thanks

**Pull request merged**